### DISCUSSION TITLE
Deleting items from File -> Open recent menu

### DISCUSSION THREAD

**Discussion by edgndg** [artisan-roaster-scope/artisan#1061] [Q&A] [answered]
Curious whether there is a place where these items are stored?  My menu options default to roasts from years ago (eg 20 roasts from 2019) -- I have done well over 100 since.  Have learned a lot since then ;-) Would like to just wipe them out and allow to repopulate.  FWIW I run on Linux Mint, currently upgraded to Artisan 2.8.2.

**Comment by roasterdave** [comment]
Can you confirm that after opening a file from the menu File>> Open..., the file does not appear at the top of the File>> Open Recent list?

**Reply by edgndg** [reply]
Hi Dave -- Thanks for response.  Your question prompted me to go back in and try a couple of things (note -- not connected to my phidget in case this is relevant).

1.  When I open Artisan, then open an existing profile, it does indeed appear at top of list.  This was a bit of "news" to me.  I did this yesterday.  I did it a couple of times to confirm.
2.   When I went back today, those entries are gone and it has reverted to what I have seen before (only very old entries).  See attached screenshot.
![Screenshot from 2022-12-27 10-46-03](https://user-images.githubusercontent.com/39470823/209689662-cd076a00-1019-4edc-aa51-767421986f6e.png)

3. I will note (in case relevant) that I typically open Artisan and run it without loading a profile but tracking temps for a warm up process.  I only get to opening profiles once I am ready to start an actual roast.

**Comment by roasterdave** [comment, marked as answer]
Are you closing Artisan from within the app as opposed to killing the process?
Is anything write protected in the path to ~/.config/artisan-scope/Artisan.conf?
What version of Mint?

**Reply by edgndg** [reply]
Sorry for delay.  Not closing the process, closing from within Artisan.  I run Linux Mint 20.2.

Your Q basically sent me to the right place.  The config file was set with root permissions (not something I would have done, must have occurred in the set up?)  I have changed to personal permissions and seems to be working now.

WRT to the items in the list, those roasts were a long time ago before various Mint and Artisan upgrades.  It is possible (I suppose) that Artisan was being run in SU mode...

Appreciate you sticking with me and pointing me to right file etc...
